src/App.jsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import './App.css'
+import play from './access/paly.svg'
+import pause from './access/pause.svg'
+import resetTime from './access/time.svg'
+import arrow from './access/arrow.svg'
+import {connect} from "react-redux";
```

Comment:
Don't mix imports. When codebase grows it's getting hard to read.

Instead of doing this thing systematically, by order, make convention for yourself. E.g. npm packages go first and then you modules go